debian/rules

```diff
@@ -12,6 +12,9 @@ OUR_GOPATH := $(CURDIR)/.gopath
 export GOPATH := $(OUR_GOPATH)
 export GOCACHE := $(CURDIR)/.gocache
 
+# https://blog.golang.org/go116-module-changes (TODO figure out a new solution for Go 1.17+)
+export GO111MODULE := auto
```

Comment:
:see_no_evil: 

Since we're using the `vendor/` directory, we might be able to get away with a mostly-empty `go.mod` and `-mod=vendor` :grimacing: (as a potential resolution to that `TODO`, to be clear, not something I'm suggesting for here)

Reply:
Right, we can explore this later. I am surprised this is still working with Golang 1.17, upstream mentioned it wouldn't work ("Go 1.17 will ignore GO111MODULE").